backend/src/routes/api/s3/files.ts

```diff
@@ -0,0 +1,256 @@
+import express = require("express");
```

Comment:
This is a new edition that adds support for reading raw files from an S3. Not sure if migrating this is totally necessary.